liquibase-couchbase/src/test/java/liquibase/ext/couchbase/database/CouchbaseConnectionTest.java

```diff
@@ -84,4 +108,31 @@ void Should_throw_error_when_open_connection_with_invalid_params() {
                 .withMessage("Could not open connection to database: %s", DB_URL);
     }
 
+    @Test
+    @SneakyThrows
+    void Should_catch_and_wrap_exception_when_cluster_cant_connect() {
+        try (MockedStatic<Cluster> mockedStatic = Mockito.mockStatic(Cluster.class)) {
+            mockedStatic.when(() -> Cluster.connect(anyString(), any()))
+                    .thenThrow(new RuntimeException("Mocked"));
+            Properties driverProperties = new Properties();
+            driverProperties.setProperty("user", "user");
+            driverProperties.setProperty("password", "password");
+
+            assertThatExceptionOfType(DatabaseException.class)
+                    .isThrownBy(() -> connection.open(DB_URL, driver, driverProperties))
+                    .withMessage("Could not open connection to database: %s", DB_URL);
+
+            mockedStatic.verify(() -> Cluster.connect(anyString(), any()));
+        }
+    }
+
+    @Test
+    void Should_support_correct_url() {
+        assertThat(connection.supports("couchbase://127.0.0.1")).isTrue();
+    }
+
+    @Test
+    void Should_not_support_incorrect_url() {
+        assertThat(connection.supports("MOCKED://127.0.0.1")).isFalse();
```

Comment:
Maybe `NotCouchbase` instead of `MOCKED`?

Reply:
done